casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -538,17 +538,23 @@ class MultiParentCasperImpl[F[_]: Sync: Concurrent: Log: Time: SafetyOracle: Las
                                equivocations.flatMap(_.equivocationDetectedBlockHashes)
                              }
                            }
-      depsInEqTracker = allDeps.filter(equivocationHashes.contains)
+      depsInEqTracker  = allDeps.filter(equivocationHashes.contains)
+      depsInProcessing <- blockProcessingState.get.map(_.processing)
+      depsEnqueued     <- blockProcessingState.get.map(_.enqueued)
```

Comment:
I must note that this `enqueued` is introduced preliminary. In current dev branch casper is locked, and we try to acquire lock only when all dependencies are in DAG, which means in single parent version `enqueued` is always empty.